packages/core/src/quote/Quote.ts

```diff
@@ -104,22 +104,22 @@ export async function createAttesterSignedQuote(
  *
  * @param quote The object which to be verified.
  * @param options Optional settings.
- * @param options.resolver DidResolver used in the process of verifying the attester signature.
+ * @param options.resolve Resolve function used in the process of verifying the attester signature.
```

Comment:
I think we should mention, that it is for resolving Dids. 
Or maybe we should rename the parameter to reflect that.
(This comment applies to all functions taking a resolve function)